ibexa/dam-user/4.6/manifest.json

```
@@ -0,0 +1,9 @@
+{
+    "aliases": [],
+    "bundles": {
+        "Ibexa\\Bundle\\DamUser\\IbexaDamUserBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    }
```

This probably can be 
```suggestion
  "aliases": [],
  "bundles": {}
```
as you don't have anything to copy there. Bundle should be added OOTB for all AFAIK. See that the most packages are specified like that.